### ISSUE TITLE
OIDC Backchannel logout token contains invalid claims

### ISSUE THREAD

**Issue by isuruhettiarachchi** [wso2/product-is#10864]
**Describe the issue:**
In the OIDC backchannel logout token, `events` claim required to be sent with a member name `http://schemas.openid.net/event/backchannel-logout` with the corresponding member value as empty JSON object { }. Current behavior send a claim `event` that does not have any members. 
e.g.
```
  {
   "iss": "https://server.example.com",
   "sub": "248289761001",
   "aud": "s6BhdRkqt3",
   "iat": 1471566154,
   "jti": "bWJq",
   "sid": "08a5019c-17e1-4977-8f42-65a12843ea02",
   "event": {}
  }
```

**How to reproduce:**
- Setup and orchestrate a OIDC backchannel logout and trace the logout request contents.

**Expected behavior:**
Logout request token required to have a claim `events` with a member `http://schemas.openid.net/event/backchannel-logout` with the corresponding member value as empty JSON object { }.
e.g.
```
  {
   "iss": "https://server.example.com",
   "sub": "248289761001",
   "aud": "s6BhdRkqt3",
   "iat": 1471566154,
   "jti": "bWJq",
   "sid": "08a5019c-17e1-4977-8f42-65a12843ea02",
   "events": {
     "http://schemas.openid.net/event/backchannel-logout": {}
     }
  }
```
**Environment information**
 - Product Version: 5.10

**Comment by piraveena** [comment]
This issue is already reported here https://github.com/wso2/product-is/issues/7680 and fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1515 PR. The fix is available with 5.12.0-m4 pack.

**Comment by piraveena** [comment]
Closing this issue since it is already reported and fixed with the latest released pack.
